tests/platform_tests/test_reboot.py

```diff
@@ -94,16 +94,26 @@ def check_interfaces_and_services(dut, interfaces, xcvr_skip_list, reboot_type =
         logging.info("Wait {} seconds for all the transceivers to be detected".format(MAX_WAIT_TIME_FOR_INTERFACES))
         result = wait_until(MAX_WAIT_TIME_FOR_INTERFACES, 20, 0, check_all_interface_information, dut, interfaces,
                             xcvr_skip_list)
-        assert result, "Not all transceivers are detected or interfaces are up in {} seconds".format(
-            MAX_WAIT_TIME_FOR_INTERFACES)
+        if dut.has_sku:
+            assert result, "Not all transceivers are detected or interfaces are up in {} seconds".format(
+                MAX_WAIT_TIME_FOR_INTERFACES)
+        else:
+            if not result:
+                logging.warn("Not all transceivers are detected or interfaces are up in {} seconds, but the required hwsku.json is not present either".format(MAX_WAIT_TIME_FOR_INTERFACES))
```

Comment:
Can you try use pytest.xfail() instead? This way we know the test failed as expected.

Reply:
I have replaced my modifications with a xfail.

Reply:
@Xichen96 i think you have still not pushed your changes here

Reply:
Sorry, I meant to push it right after, but then I noticed my access token had just elapsed. Took me a while to set it up again.